server/src/auth/dtos/resource.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { InputType, Field, registerEnumType } from '@nestjs/graphql';
+import { RequestMethod } from '@nestjs/common';
+
+registerEnumType(RequestMethod, { name: 'RequestMethod' });
```

Comment:
Nice

Reply:
At first I was making a custom enum, then realized that it probably already existed lol